src/parse/errors.rs

```diff
@@ -446,6 +446,15 @@ impl Error {
         safe_exit(0)
     }
 
+    /// Prints the error to `stderr`
+    pub fn print(&self) {
+        if self.use_stderr() {
+            self.message.print().expect("Error writing Error to stderr");
```

Comment:
Please exposing Error rather than panic.

Reply:
Ok. Is it good `pub fn print(&self) -> io::Result<()>`

Reply:
I'm okay with it. cc @pksunkara